templates/user-data.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+. /etc/environment
+USERDATA_FILE={{.MachineType}}
+
+/usr/bin/rkt run \
+    --net=host \
+    --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \
+    --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \
+    --trust-keys-from-https \
+    quay.io/coreos/awscli:latest -- aws s3 --region {{.Region}} cp s3://{{.S3Bucket}}/{{.ClusterID}}/cloudconfig/$USERDATA_FILE /var/run/coreos/temp.txt
```

Comment:
I am not sure about the reliability of this `latest` tag. I think it would make sense to use a more proper version which might not be as often overwritten as the latest tag.